### PR TITLE
added option to select database, mariadb or mysql

### PR DESCRIPTION
**Pull request by ethancedwards8**
Adding option to use either mariadb or mysql database. Doesn't change defaults.

### CODE REVIEW

File: public/v2/apps/wordpress.json [hunk 1]
```diff
@@ -59,10 +59,17 @@
             "validRegex": "/^([^\\s^\\/])+$/"
         },
         {
-            "id": "$$cap_mysql_version",
-            "label": "MySQL Version",
+            "id": "$$cap_db_type",
+            "label": "Database Type",
+            "defaultValue": "mysql",
+            "description": "You can either choose mariadb or mysql, you need to change the version according to which DB is selected. It is case sensitive.",
+            "validRegex": "/.{1,}/"
```
Comment (githubsaturn) [inline public/v2/apps/wordpress.json:66]: Let's change the regex to only allow mysql and mariadb

Comment (ethancedwards8) [reply public/v2/apps/wordpress.json:66]: I'm not very in regex, any clues on how to do that?

Comment (githubsaturn) [reply public/v2/apps/wordpress.json:66]: Maybe something like this?
```
"/^(mysql|mariadb)$/"
```

Comment (ethancedwards8) [reply public/v2/apps/wordpress.json:66]: I'll try it out.

**Comment by ethancedwards8** [comment]
That seems to have worked

File: public/v2/apps/wordpress.json (hunk 1 above)
Comment (ethancedwards8) [reply public/v2/apps/wordpress.json]: That worked

**Pull request merged**